src/components/Alert/Alert.css

```
@@ -1,8 +1,24 @@
 .Alert {
   user-select: none;
-  color: var(--text_primary);
+  color: var(--text_primary, var(--vkui--color_text_primary));
   position: relative;
+  min-width: 280px;
   max-width: 416px;
+  margin: auto 24px;
+  box-shadow: var(--vkui--elevation4);
+  border-radius: var(--vkui--size_border_radius_paper--regular);
+  animation: vkui-animation-android-alert-intro 0.2s var(--android-easing);
+}
+
+.Alert--desktop {
+  width: 416px;
```

```suggestion
  width: 100%;
```

Учитывая, что выставлен `max-width: 416px` мб так? Чтобы не дублировалось значение

---

✅